sonic-net/sonic-buildimage
# Update docker-sonic-vs to Bookworm

Bookworm SWSS Layer is available. Upgrade docker-sonic-vs to bookworm.

##### Work item tracking
- Microsoft ADO **(number only)**: 30284948

#### How I did it

Update `Dockerfile` and `docker-sonic-vs.mk` to use SWSS Bookworm image/layer.

#### How to verify it

Build image and manually verify contents of `/etc/os-release` is `Bookworm` by loading `docker-sonic-vs.img`.

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)



Not applicable

#### Description for the changelog

[docker-sonic-vs]: Update to Bookworm

* Bookworm SWSS Layer is available. Upgrade docker-sonic-vs to bookworm.
